### PR TITLE
Updated the .codeclimate.yml file

### PR DESCRIPTION
**Pull request by opg7371**
Removed the tests directory as you recommended.
Updated the .codeclimate.yml file
Please review ...:)
#328

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 exclude_paths:
 - vendor/**/*
 - ZeroBin/**/*
-- tests/**/*
```
Comment (smcgregor) [inline .codeclimate.yml:4]: What is the difference here between two `**` and one `*`?

**Comment by smcgregor** [comment]
I think this is your third pull request making similar changes to the codeclimate file. You should be updating a single pull request, or at least closing the pull requests you open that are no longer valid.

**Comment by opg7371** [comment]
yes i will close other pull requests......I created this since you hinted me to remove a line which was not needed.Is there anything else that i should add ??
Should i reference this to the pull request that is still open??

File: .codeclimate.yml (hunk 1 above)
Comment (opg7371) [inline .codeclimate.yml:4]: I don't know exactly but if have to guess,then they both are pretty same.I will update them

Comment (smcgregor) [inline .codeclimate.yml:4]: Please don't guess. You can't ask for something to be merged if you don't know what the code does.

Comment (opg7371) [inline .codeclimate.yml:4]: I researched on this thing...they (\* & *\* ) are both same and does same work in this case for sure

**Comment by opg7371** [comment]
The new pull request is present at....#355
So i am closing this issue

File: .codeclimate.yml (hunk 1 above)
Comment (smcgregor) [inline .codeclimate.yml:4]: OK, thanks.

**Pull request closed without merging**